content/file-and-directory-naming/general/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+_db_id: 768
+content_type: topic
+ready: true
+title: File and directory naming
+---
+
+## Naming files and directories
+
+You can organize your files and directories in a logical and consistent manner by using naming conventions. The use of naming conventions helps to ensure the correct information can be located, identified, and retrieved from a file system quickly, as well as that it is stored in a secure location.
+
+## What should I consider when naming files and directories
+
+New files and directories (including third-party frameworks) should adhere to the recommended naming conventions, but if a library already exists that
+has a different style, internal consistency should take precedence.
+
+## A common pitfall to avoid
+
+Many times, people start by writing code that's messy and not very organized. They only try to make it look neater after it works. But this can cause problems because it might make mistakes, and they might miss important things. It's better to start with good coding habits from the beginning, even if it seems a bit slower at first. Once you get used to it, you'll end up being better and faster at it.
```

Comment:
the 3rd sentence is a bit confusing, I think it's a mistakes. Should it not be: `But this can cause problems because they might make mistakes, and they might miss important things`